### DISCUSSION TITLE
Need help with the notification settings

### DISCUSSION THREAD

**Discussion by Ukchris79** [djdd87/SynoAI#135] [Q&A]
All, I need some help please

I have never used push bullet or webhooks, what is the best way to set something up for the notifications?

Thanks

Chris

**Comment by djdd87** [comment]
Hi Chris, everything should be in the readme for all the supported notification types. Personally I use Telegram. Let me know if you have any specific questions.

**Reply by Ukchris79** [reply]
I assumed that this solution would be completely self-hosted, doesn't using apps like telegram slow down the system?

**Reply by Ukchris79** [reply]
I have just realised my mistake, I assumed that the notification fired the video capture rather than just a notification to the user.

**Reply by djdd87** [reply]
You can do that by using a webhook that you define on SSS.
